crates/fluvio-connector/src/config.rs

```diff
@@ -0,0 +1,50 @@
+use serde::Deserialize;
+use std::collections::BTreeMap;
+use std::path::PathBuf;
+use std::fs::File;
+use std::io::Read;
+
+use fluvio_controlplane_metadata::connector::ManagedConnectorSpec;
+use crate::error::ConnectorError;
+
+#[derive(Debug, Deserialize)]
+pub struct ConnectorConfig {
+    #[serde(default = "String::new")]
+    name: String,
+    #[serde(rename = "type")]
+    type_: String,
+    topic: Option<String>,
```

Comment:
topic should be mandatory?

Reply:
Or it could default to the `type`. Could go either way.

Reply:
We have decided to make this mandatory.